cypress/support/commands.ts

```diff
@@ -29,3 +29,6 @@ Cypress.on("uncaught:exception", (err, runnable) => {
     // failing the test
     return false;
 });
+
+import 'cypress-file-upload';
+
```

Comment:
import should go at before the Cypress.on call

Reply:
all done @holgiosalos